www/js/splash/customURL.ts

```diff
@@ -0,0 +1,22 @@
+type UrlComponents = {
+    [key : string] : string
+}
+
+type OnLaunchCustomURL = (rawUrl: string, callback: (url: string, urlComponents: UrlComponents) => void ) => void;
+
+
+export const onLaunchCustomURL: OnLaunchCustomURL = (rawUrl, handler) => {
```

Comment:
I'm confused. where is this called from? I only see code deleted from `ngApp` - I don't see anything added there.

Reply:
As Jack mentioned, `onLaunchCustomURL` is no longer called after the onboarding flow changed. Nevertheless, we think it's still valuable to keep this code.